inspector/src/components/actionTabs/tabs/propertyGrids/animations/addAnimation.tsx

```diff
@@ -64,10 +145,37 @@ export class AddAnimation extends React.Component<
         dataType = Animation.ANIMATIONTYPE_COLOR4;
         break;
     }
-
     return dataType;
   }
 
+  getTypeAsString(type: number) {

```

Comment:
why not return in each case instead ? would save a bit of code.

Reply:
You mean like  "case Animation.ANIMATIONTYPE_FLOAT: return 'Float' "?

Reply:
yup but it is really only a nit nothing to worry to much